lib/compress/zstd_compress.c

```diff
@@ -2352,7 +2352,7 @@ static size_t ZSTD_buildSeqStore(ZSTD_CCtx* zc, const void* src, size_t srcSize)
                                        ms, &zc->seqStore,
                                        zc->blockState.nextCBlock->rep,
                                        src, srcSize);
-            assert(ldmSeqStore.pos == ldmSeqStore.size);
+            //assert(ldmSeqStore.pos == ldmSeqStore.size);
```

Comment:
This may no longer be necessarily true, since `ZSTD_BtGetAllMatches()` might never get called at the absolute byte position of an LDM.